src/bot.js

```diff
@@ -69,6 +75,58 @@ function run() {
     )
 }
 
+function sendSlackMessageWithBirthDays(deadline) {
+    return (
+        // Se obtienen los cumpleaños desde un spreadsheet
+        gsheets
+            .getWorksheet(
+                process.env.REACT_APP_SPREADSHEET_ID,
+                process.env.REACT_APP_WORKSHEET_ID
+            )
+            .then(response => response.data)
+            // se filtran los cumpleaños del dia de hoy
+            .then(birthdays => birthdays.filter(
+                birthday => deadline.format('DD/MM') === moment(birthday['Fecha'], 'DD/MM').format('DD/MM')
+            ))
+            // si hay alguno, se formatea el mensaje
+            .then(birthdaysOfTheDay => {
+                if (!birthdaysOfTheDay.length) {
+                    return Promise.reject('No hay cumpleaños el dia de hoy. :(')
+                }
+                return birthdaysOfTheDay.reduce((message, birthday) => {
+                    return (
+                        message +
+                        `>*${birthday['Nombre']}* - @${birthday['Usuario de Slack']}\n\n`
+                    )
+                }, '')
+            })
+            .then(message => `${BIRTHDAY_HEADER}${message}${BIRTHDAY_FOOTER}`)
+            .then(message => {
+                const messageOptions = {
+                    channel: process.env.CHANNEL,
+                    // party emoji
+                    icon_emoji: 'https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/google/146/face-with-party-horn-and-party-hat_1f973.png',
```

Comment:
Esta imágen (que creo que deberímos buscar una más flashera jajaja) tenemos que tenerla en la carpeta `assets` en la raíz del repo y agregarla acá https://github.com/meetupjs-ar/calendar-bot/blob/master/bots.json

La línea de código para usar específicamente un nuevo bot (asumiendo que se llama `birthdaysBot`) sería así:

```suggestion
                    icon_emoji: require(process.env.BOTS_PATH).bots.birthdaysBot,
```

Obvio que `require(process.env.BOTS_PATH).bots` deberíamos meterlo en una constante y toda la bola, pero es para ilustrar la sugerencia.